React-insta-clone

mikeyjwilliams opened the pull request:
1. worked on nav bar
1. worked on PostContainer
1. note not quite done with building out PostContainer
hoping to complete this between tonight and tomorrow afternoon

mikeyjwilliams committed:
feat(instagram-clone) + beginning instagram-clone files

mikeyjwilliams committed:
fresh react start =>
- serviceworker
- svg logo

mikeyjwilliams committed:
mv(dummy-data.js) + dummy-data => src folder

mikeyjwilliams committed:
test(App.js) + import dummy-data
+ console.log dummy-data test

mikeyjwilliams committed:
app.js => remove react landing page

mikeyjwilliams committed:
style(App.[css | js])  + .ig-layout
css => + flex & space-around styling

mikeyjwilliams committed:
plan(App.js) wrot out plan for plan of attack below export.

mikeyjwilliams committed:
style add

mikeyjwilliams committed:
constructor added for state

mikeyjwilliams committed:
add to PostContainer dir

mikeyjwilliams committed:
format data

mikeyjwilliams committed:
go back to simple for now.

mikeyjwilliams committed:
build out component PostContainer.js

mikeyjwilliams committed:
; => PostContainer.js

mikeyjwilliams committed:
move components out of src directory

mikeyjwilliams committed:
moved into src

mikeyjwilliams committed:
+ fontawesome for icons

mikeyjwilliams committed:
yarn lock

mikeyjwilliams committed:
+ SearchBar file
+ fontawesome icons
+ building out searchbar

mikeyjwilliams committed:
Built out search bar

mikeyjwilliams committed:
App.js + SearchBar component added

mikeyjwilliams committed:
mod(SearchBar.css) + margin to bottom of style

mikeyjwilliams committed:
feat(PostContainer.js) + thumbnail
+ username
+ imageUrl

mikeyjwilliams committed:
style(PostContainer.css) styles for layout so far

mikeyjwilliams committed:
using dummydata in App.js

mikeyjwilliams committed:
add svg icons to package.json

mikeyjwilliams committed:
style(PostContainer) center insta-post

mikeyjwilliams committed:
feat(PostContainer) + imports
+ hear to interaction section

mikeyjwilliams committed:
style(SearchBar.css) rm sizes to icons

mikeyjwilliams committed:
feat(SearchBar.js) + size on FontAWesome components

mikeyjwilliams committed:
Feat(PostContainer.js) + proptypes =>igData
+ insta comments && likes
+ import proptypes

mikeyjwilliams committed:
add files to dir

mikeyjwilliams committed:
mod(PostContainer.js) from Required to normal proptypes

mikeyjwilliams committed:
build component

mikeyjwilliams committed:
component container for map

mikeyjwilliams committed:
import commentContainer and use

mikeyjwilliams committed:
style(PostContainer) + style for heart &&
likes in post

mikeyjwilliams committed:
style(Comments.css) + flex
+ margin-r username

mikeyjwilliams committed:
feat(Comment.js) + username
+ text

mikeyjwilliams committed:
import NewComment
+ NewComment component

mikeyjwilliams committed:
new component

mikeyjwilliams committed:
style(Comment.css) comment styles

mikeyjwilliams committed:
+ need to add more proptypes

bstrellis reviewed (commented):
Day 1:
- [] Follows Lambda Git Flow // Remember to DM me your collab invite and add me as a reviewer, please.
- [x] Frequent, clear, and descriptive commits // Some of your commits are difficult to understand.  Imagine returning to this in a few weeks: "; => PostContainer.js".  Difficult to know what that would mean, huh?  Especially for another developer! So be sure that your commits are going to be meaningful when you come back to the project later.
- [x] No errors in the console
- [x] Code is sufficiently commented
- [x] Code is organized and easy to read
- [] MVP Complete // Add in propTypes for comment.  Please let me know if you need help with this.  
- [] Stretch

Day 2:
- [x] Follows Lambda Git Flow
- [x] Frequent, clear, and descriptive commits
- [x] No errors in the console
- [x] Code is sufficiently commented
- [x] Code is organized and easy to read
- [x] MVP Complete // Search bar was close.
- [] Stretch

Michael, good job hacking away at this.  You are a coding hero! This was an extremely tough assignment.  You're on the right track to think about organization first and foremost when dealing with React.

mikeyjwilliams committed:
style(Comment.css) + remove border
add comment

mikeyjwilliams committed:
style(SearchBar.css) + modname => input-search | searchBar
for uniformity

mikeyjwilliams committed:
format(SearchBar.js) .searchBar => .input-search

mikeyjwilliams committed:
removed packages from wrong zone

mikeyjwilliams committed:
test(package.json + fontawesome pro
2019-06-14-10:42:54

mikeyjwilliams committed:
style(searchbar.css) line between logo and brand name

mikeyjwilliams committed:
style(Comment.css) styles moved

mikeyjwilliams committed:
feat(Comment.js) + className to p

mikeyjwilliams committed:
style(PostContainer.css) - border

mikeyjwilliams committed:
Merge branch 'michael-williams' of https://github.com/mikeyjwilliams/React-Insta-Clone into michael-williams

mikeyjwilliams committed:
format code w/ prettier

mikeyjwilliams committed:
feat(googleFont) + google font api link

mikeyjwilliams committed:
feat(Comment.js) + proptypes

mikeyjwilliams committed:
style(SEarchBar.css) + font Lobster
for instagram brand name

mikeyjwilliams committed:
feat(NewComment.js) + input inside form

mikeyjwilliams committed:
feat(NewComment.js) + onChange TextEvent
+ onSubmit eventHandler
+ rework too class component

mikeyjwilliams committed:
feat(NewComment.js) + state with beginning state / maybe change to CDM
+ mv event handlers into class
+ this  too onSubmit eventHandler
+ this too onChande textEvent

mikeyjwilliams committed:
rework(NewComment.js) + props for state
+ functional component
- class componenent
- event handlers in class
having its own state made no sense so it it away. mod back to functional component
with props for needs with state

mikeyjwilliams committed:
feat(CommentContainer.js) + submit, change, value given props for NewComment component

mikeyjwilliams committed:
docs(App.js) - tooks comments from bottom away

mikeyjwilliams committed:
fix conflict(NewComment.js)

mikeyjwilliams committed:
lockpackage w/
free-brands-icons

mikeyjwilliams committed:
feat(App.js) + class extendApp.js component
+ using state
+ submits coming up... not sure if they should reach this level though?

mikeyjwilliams committed:
format(CommentContainer.js) + spaces added

mikeyjwilliams committed:
feat(PostContainer.js) + convert to React.Component
+ has state
+ changeSubmit
+ commentValue
on this component.

mikeyjwilliams committed:
rm(App.js) submits from app.js

mikeyjwilliams committed:
feat(PostContainer.js) + starter state

mikeyjwilliams committed:
Merge branch 'michael-williams' of https://github.com/mikeyjwilliams/React-Insta-Clone into michael-williams

mikeyjwilliams committed:
test(CommentContainer.js)  + alter from what was thought needed to original process.

mikeyjwilliams committed:
fontawesome pulls to node modules

mikeyjwilliams committed:
Merge branch 'michael-williams' of https://github.com/mikeyjwilliams/React-Insta-Clone into michael-williams

mikeyjwilliams committed:
docs(NewComment.js) comment on end of line for what it is

mikeyjwilliams committed:
i believe this one needs state CommentContainer.js

mikeyjwilliams committed:
feat(NewComment.js) change event name format

mikeyjwilliams committed:
not sure if right investigating
feat(CommentContainer.js)
+ add state to container

mikeyjwilliams committed:
redo(App.js) changing where form data is pushed to state

mikeyjwilliams committed:
feat(CommentContainer.js) + thoughts on might use as state

mikeyjwilliams committed:
format(NewComment.js) format names

mikeyjwilliams committed:
style(PostContainer.css) mv styles to another page

mikeyjwilliams committed:
format(PostContainer.js) change component

mikeyjwilliams committed:
style(ListesSection.css) add styles for Likes

mikeyjwilliams committed:
feat(LisesSection.js) + function for likes

mikeyjwilliams committed:
rid of event listener in app.js

mikeyjwilliams committed:
feat(PostContainer.js) react component convert

mikeyjwilliams committed:
feat(CommentContainer.js) functional component

mikeyjwilliams committed:
feat(CommentContainer.js) + data pass in by props
- rm this key word to props

mikeyjwilliams committed:
format(NewComment.js) / format names

mikeyjwilliams committed:
feat(PostContainer.js) + event handlers
+

mikeyjwilliams committed:
props(Comment.js) + proptypes

mikeyjwilliams committed:
proptypes(Comment.js) +shape of proptypes

mikeyjwilliams committed:
nameMod(CommentContainer.js) + mod component to CommentForm

mikeyjwilliams committed:
update to CommentForm.js from NewComment.js

mikeyjwilliams committed:
feat(PostContainer.js) + change names of state
+ alter addCommentHandler inside
+ alter commentHandler inside

mikeyjwilliams committed:
rework(commentForm.js) strip down component to only form

mikeyjwilliams committed:
format(App.js) + format names for consistency!

mikeyjwilliams committed:
feat(CommentContainer.js) + styling for container = moved from CommentForm.js
e.g fontawesome stuff
+ changed name of incoming data prop
for naming consistency

mikeyjwilliams committed:
format(PostContainer.js) + remove extra spaces
+ data name change for consistency
+ comments in event handler
- remove console.logs

mikeyjwilliams committed:
format(LiskesSection.js) + change data name
for naming consistency

mikeyjwilliams committed:
package(Package.json) + reactstrap
+ bootstrap

mikeyjwilliams committed:
TESTING (App.js) fontawesome with react strap

mikeyjwilliams committed:
feat(index.js) bootstrap added

mikeyjwilliams committed:
style(PostHeader.css) + post header styles added to own sheet

mikeyjwilliams committed:
feat(PostHeader.js) functional component built

mikeyjwilliams committed:
style(PostContainer.css) - styles for single Post

mikeyjwilliams committed:
feat(PostContainer.js) + PostHeader component
+ format LikeSection Component -> future work

mikeyjwilliams committed:
feat(LikeSection.js) / using fontawesome in
different way to showcase emoji icons.
-- goal to showcase thin lined ones in future.

mikeyjwilliams committed:
style-name(LiskesSection.css) + mof style main style name

mikeyjwilliams committed:
fix(App.js) reduce none needed prop in
CDM ()

mikeyjwilliams committed:
fix(PostHEader.js) - remove this from props
not a class just a functional component

mikeyjwilliams committed:
remove files

mikeyjwilliams committed:
style(PostContainer.css) styles have moved to other component style sheets

mikeyjwilliams committed:
format(PostContainer.js) format code

mikeyjwilliams committed:
format(LikeSection.css) format code

mikeyjwilliams committed:
format(LikesSection.js) format code

mikeyjwilliams committed:
code fomart LikesSection.js

mikeyjwilliams committed:
TEST(App.js) TESTING + fontawesome with bootstrap

mikeyjwilliams committed:
mod(LikeSection.js) + icons are appearing correctly

mikeyjwilliams committed:
docs(App.js)  - remove comments
+ add a loading if datat doesnt load automatically.

mikeyjwilliams committed:
style(App.css) + center loading text

mikeyjwilliams committed:
feat(PostContainer.js) + 'likes' from instagramData to this.state

mikeyjwilliams committed:
format(LisesSection.js) + format code
to be more readable

mikeyjwilliams committed:
feat(PostContainer.js) + hartLikesHandler
event handler started

mikeyjwilliams committed:
style(LikesSection.css) + fix heart button to look like page

mikeyjwilliams committed:
feat(LikesSEction.js) / change to button from div around heartIcon
+ type=button
+ onclick added

mikeyjwilliams committed:
mod(PostContainer.js) / rename eventhandler for button for heart likes

mikeyjwilliams committed:
rename(LikesSEction.js)

mikeyjwilliams committed:
likes datat to point to comments

mikeyjwilliams committed:
revert due to confusion in code

This reverts commit efe5bbef609d1ca77877dcfd2aef25533ae55a41.

mikeyjwilliams committed:
format(LikesSection.js) format code

mikeyjwilliams committed:
redo(PostContainer.js) + reworking from slowly from the ground up bit by bit.

mikeyjwilliams committed:
fix postContainer

mikeyjwilliams committed:
delete old file

mikeyjwilliams committed:
format page App.js

mikeyjwilliams committed:
update Post.css

mikeyjwilliams committed:
built Post.js

mikeyjwilliams committed:
map out Post component

mikeyjwilliams committed:
format LikeSection.css

mikeyjwilliams committed:
build out LikesSEction

mikeyjwilliams committed:
update font aweseom

mikeyjwilliams committed:
back to original version from previous day

mikeyjwilliams committed:
committing back to sundays version due to burning down the progress done today

mikeyjwilliams committed:
add fontawesome icon for heart

mikeyjwilliams committed:
format (LikesSection.js) + format
+ using fontawesome icon
/ altered click evnt name

mikeyjwilliams committed:
update(Post.js) removed state from this place for the heart increase
-> felt it was better to do it further up.

mikeyjwilliams committed:
Feat(PostContainer.js) + increaseLikes
event handler working
and adding likes to each  person
+ removed console logs

mikeyjwilliams committed:
got rid of forawesome icons using regular
fontawesome icons

mikeyjwilliams committed:
feat(SearchBar.js) + searchbar event handler
on props

mikeyjwilliams committed:
remove unused or old files

mikeyjwilliams committed:
renamed file directories

mikeyjwilliams committed:
changed container name

mikeyjwilliams committed:
put frowns in comments this was annoying.
